bouncer/util.py

```diff
@@ -44,15 +45,31 @@ def parse_document(document):
     annotation_id = document["_id"]
     annotation = document["_source"]
 
+    # And that annotations always have "group" and "shared"
+    group = annotation["group"]
+    shared = annotation["shared"]
+
+    text = annotation.get('text', 'Follow this link to see the annotation on the original page.')
+
     document_uri = None
+    quote = None
 
     try:
         targets = annotation["target"]
         if targets:
             document_uri = targets[0]["source"]
+            if 'selector' in targets[0]:
```

Comment:
Got it, thanks. So does `dict.get(key)` pretty much obsolete `dict.get[key]` at this point?